lib/asciidoctor/js/opal_ext/graalvm/file.rb

```diff
@@ -0,0 +1,7 @@
+class File
+
+  def self.read(path)
+    `IncludeResolver.read(path)`
```

Comment:
Maybe I'm doing something wrong but the following code throws an exception:

```js
var Paths = Java.type('java.nio.file.Paths');
```

```
Exception in thread "main" TypeError: Java Interop is not available
```
Reference: https://github.com/graalvm/graaljs/blob/master/docs/user/JavaInterop.md#java-interoperability-features

To workaround this issue, I'm using a technique described in the GraalVM documentation: https://www.graalvm.org/docs/graalvm-as-a-platform/embed/#access-java-from-graal-languages

In my Java application I declare a class.
This class can be passed to the JavaScript context using `Polyglot.import` (see `spec/graalvm/AsciidoctorConvertWithGraalVM.java`)